build/charts/antrea/conf/antrea-agent.conf

```diff
@@ -23,6 +23,9 @@ featureGates:
 # Enable traceflow which provides packet tracing feature to diagnose network issue.
 {{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "Traceflow" "default" true) }}
 
+# Enable PacketCapture feature which provides packets capture feature to diagnose network issue.
+{{- include "featureGate" (dict "featureGates" .Values.featureGates "name" "PacketCapture" "default" false) }}
```

Comment:
remember to add the new feature gate to `docs/feature-gates.md`

Reply:
updated.